server/videoSplit/videoSplitConcept.py

```diff
@@ -0,0 +1,154 @@
+import cv2
+from skimage.metrics import structural_similarity
+from moviepy.video.io.ffmpeg_tools import ffmpeg_extract_subclip
+import os
+
+# Creates new directory for output files if no directory is set
+def create_new_target_directory(filename):
+    print(__file__)
+    os.makedirs(os.path.dirname(__file__) + "/" + os.path.basename(filename), exist_ok=True)
```

Comment:
This should be done using os.path.join (especially as os.path is already being used).